salt/utils/crypt.py

```diff
@@ -103,8 +124,8 @@ def reinit_crypto():
         child processes after using os.fork()
 
     """
-    if HAS_CRYPTO:
-        Crypto.Random.atfork()
+    if HAS_CRYPTODOME or HAS_CRYPTO:
```

Comment:
This could still run `atfork` if both m2crypto and cryptodomex are installed. We only want it to run when there is no m2crypto. In other words, skip atfork when m2crypto is installed.

Reply:
I see these are always false when m2crypto is there. Looks good upon further review :)